public/src/main/java/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -26,6 +31,7 @@ import java.util.Date
  * there is a billing issue. Check the `isActive` property.
  */
 @Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
```

Comment:
do we need this if we also have @IgnoredOnParcel?

Reply:
`jsonObject` isn't ignored though.